### DISCUSSION TITLE
How could I disable default key shortcuts when I'm editing a textedit?

### DISCUSSION THREAD

**Discussion by wangxiaochuTHU** [emilk/egui#3325] [General]
Pressing `Key::ArrowLeft` is desired to move cursor left within the textedit, but now it leads to a navigation to other widgets. 

I've tried set the textedit to be `code_editor` as follows, but this doesn't help. Am I missing any settings?
```
let resp = ui.add(
                            egui::TextEdit::singleline(&mut self.text_input)
                                .id(id)
                                .code_editor(),
                        );
```

**Comment by YgorSouza** [comment]
That's a bug. #3324

**Reply by wangxiaochuTHU** [reply]
Thanks! I'll then temporarily mask that P.R. before the bug is fixed.
